### ISSUE TITLE
documentation of APS

### ISSUE THREAD

**Issue by drjova** [cern-sis/issues-scoap3#177]
Create a detailed description of the APS publisher how we get and, store and process the data. 

- [ ] We have to have a map between each article's input and final output to understand how each field is populated.
- [ ] Default parameters and when it runs.
- [ ] How to run it manually with custom parameters (examples)
- [ ] A graphical overview of the process.

**Comment by ErnestaP** [comment]
I have some questions regarding the task:
1. We have to have a map between each article's input and final output to understand how each field is populated.
          * Article’s input - json field path and how that value is put in our final json output?
          * What should be the format of mapping: excel sheet is enough?
2. Default parameters and when it runs.
          * Default parameters if the value does not exist?
             We already discussed it, that we need to get rid off the default values in the code, since:
             If the value is required, we should not simulate fake data, but we have to raise an error, that field value is missing
             There is no sense of putting a default value in the field which is not required. We can just not have this field in the              final JSON.
3. How to run it manually with custom parameters (examples)	
          * Run harvesting with custom parameters: for example different date?

**Comment by drjova** [comment]
> 1. We have to have a map between each article's input and final output to understand how each field is populated.
>    * Article’s input - json field path and how that value is put in our final json output?
Yes map between the original input and the final output, including the fields that we caclulate.
>    * What should be the format of mapping: excel sheet is enough?
A table in this issue (markdown) should be sufficient to move it anywhere we need.

> 2. Default parameters and when it runs.
>    * Default parameters if the value does not exist?
What do you mean? 

>    We already discussed it, that we need to get rid off the default values in the code, since:
>    If the value is required, we should not simulate fake data, but we have to raise an error, that field value is missing
>    There is no sense of putting a default value in the field which is not required. We can just not have this field in the              final JSON.

What we need is which are the available parameters, which are required, what are the default values etc.

> 3. How to run it manually with custom parameters (examples)
>    * Run harvesting with custom parameters: for example different date?

Yes, an example if we want to re-run it manually with custom parameters.
